fawltydeps/packages.py

```diff
@@ -422,8 +433,14 @@ def installed_requirements(
             text=True,
             check=False,
         )
+        pip_path = (
+            venv_dir / "Scripts" / "pip.exe"
+            if platform.system() == "Windows"
+            else venv_dir / "bin" / "pip"
+        )
```

Comment:
Sometimes the multiline `if-else` expression are easier to read as an old-fashioned `if ...: ... else:` statement:
```suggestion
        if sys.platform.startswith("win"):  # Windows
            pip_path = venv_dir / "Scripts" / "pip.exe"
        else:  # Assume POSIX
            pip_path = venv_dir / "bin" / "pip"
```